api/envoy/config/common/tap/v2alpha/common.proto

```diff
@@ -14,6 +15,15 @@ option java_package = "io.envoyproxy.envoy.config.common.tap.v2alpha";
 
 // Common configuration for all tap extensions.
 message CommonExtensionConfig {
+
+  message TDSConfig {
```

Comment:
Note that @htuch has either taken "TDS" for the runTime Discovery Service. I think we are at the limits of our 3-letter acronyms. Can you maybe just call it "TapDS" or something?